### DISCUSSION TITLE
Close sunroof when locked

### DISCUSSION THREAD

**Discussion by gurmukhp** [bimmerconnected/bimmer_connected#647] [Q&A]
With the BMW i4, it's possible to have the sunroof in many states, there are 3 states, one is just to show the glass roof, then the other is to let air in at an angle and the final step is to open the sunroof.

Is it possible to always close the sunroof so it's completely closed when the car is locked? I can't seem to find anything in the sensors.

**Comment by rikroe** [comment]
Are you interested in a sensor, i.e. just knowing the current state?
Or would you like to interact with car and actually close the sunroof?

**Comment by gurmukhp** [comment]
I'd like to be able to close the sunroof if it's in any of it's opened
states.
